pkg/authz/auth_test.go

```diff
@@ -1,159 +0,0 @@
-/*
-￼Copyright 2021 Kube RBAC Proxy Authors rights reserved.
-￼
-￼Licensed under the Apache License, Version 2.0 (the "License");
-￼you may not use this file except in compliance with the License.
-￼You may obtain a copy of the License at
-￼
-￼    http://www.apache.org/licenses/LICENSE-2.0
-￼
-￼Unless required by applicable law or agreed to in writing, software
-￼distributed under the License is distributed on an "AS IS" BASIS,
-￼WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-￼See the License for the specific language governing permissions and
-￼limitations under the License.
-￼*/
-
-package authz
-
-import (
-	"context"
-	"testing"
-
-	"k8s.io/apiserver/pkg/authentication/user"
-	"k8s.io/apiserver/pkg/authorization/authorizer"
-)
-
-func TestStaticAuthorizer(t *testing.T) {
```

Comment:
missing multiple path non-resource requests.  `/heathz/detailedfailure` for instance.